align/compiler/match_graph.py

```diff
@@ -280,6 +280,12 @@ def _update_order_block_const(self,name,G1,remove_nodes,new_inst):
             G1 (graph): subckt graph
             remove_nodes (list): nodes which are being removed
         """
+
+        def _list_repace(lst, old_value, new_value):
```

Comment:
Small typo you may want to fix `_list_replace` instead of `_list_repace`. Doesn't hurt functionality so approving anyway.